x-pack/plugins/enterprise_search/server/lib/indices/create_index.ts

```diff
@@ -64,8 +68,14 @@ const defaultMappings = {
 export const createApiIndex = async (
   client: IScopedClusterClient,
   indexName: string,
-  language: string | undefined | null
+  language: string | null
 ) => {
+  await addConnector(
```

Comment:
We probably shouldn't merge this until [this discussion](https://docs.google.com/document/d/1rGL66wfSQwaOM6Ou3yFkKEOJAP4JlVpQOqr9uzTVC98/edit?disco=AAAAfIrIMH4) gets weighed in on. If it hasn't seen much discussion early next week, I'll make a separate PR that doesn't include the changes to the `create_index` API, so that the connectors API changes don't remain blocked.